asv_bench/benchmarks/benchmarks.py

```diff
@@ -148,6 +148,41 @@ def time_merge(self, shapes, how, sort):
         )
 
 
+class TimeMergeCategoricals:
+    param_names = ["shapes", "data_type"]
```

Comment:
What about "how", "sort"?

Reply:
I have an assumption that it is enough to test these parameters in the rest `merge` benchmarks. This seems to be a special case with categories.

Reply:
https://github.com/modin-project/modin/pull/5021/files#r987893073